spec/lib/upload_creator_spec.rb

```diff
@@ -43,6 +43,27 @@
         expect(File.extname(upload.url)).to eq('.png')
         expect(upload.original_filename).to eq('png_as.png')
       end
+
+      describe 'for webp format' do
+        before do
+          SiteSetting.authorized_extensions = '.webp|.bin'
+        end
+
+        let(:filename) { "webp_as.bin" }
+        let(:file) { file_from_fixtures(filename) }
+
+        it 'should not correct the coerce filename' do
```

Comment:
should not correct the coerce filename => should not coerce filename

:)

Reply:
29k is giant for a webp maybe we can make a smaller one for tests we should be able to swing a webp in 1000 bytes

Reply:
found a webp file that is 26 bytes.